### PR TITLE
Timer functionality sa

### PR DESCRIPTION
**Pull request by StevieAmb**
## Branch title

- timer-functionality-SA

## Changes made

- The code was changed so that each button can only be clicked one at a time, and no button can be double clicked. The code was also changed so that no other characters besides numbers can be entered in the minutes and seconds input fields.

## Anything Refactored

- No.

## Anything needed

- Not at the moment.

## Contributors

* Stevie Ambroise
* Alex Fritz
* Devon Wigle

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -14,16 +14,29 @@ var timerView = document.querySelector('.timer-presenter');
 var userActivityInputView = document.querySelector('.user-activity-input');
 //inputs
 var userAccomplishInput = document.querySelector('#user-accomplish');
-var userMinutesInput = document.querySelector('#minutes');
-var userSecondsInput = document.querySelector('#seconds');
+var userMinutesInput = document.querySelector('#user-minutes');
```
Comment (alexmfritz) [inline src/main.js:17]: Good catch, Devon!

File: src/main.js [hunk 2]
```diff
@@ -14,16 +14,29 @@ var timerView = document.querySelector('.timer-presenter');
...
+var userSecondsInput = document.querySelector('#user-seconds');
 //errors
 var selectionError = document.querySelector('.selection');
 var accomplishError = document.querySelector('.accomplish');
 var minutesError = document.querySelector('.minutes');
 var secondsError = document.querySelector('.seconds');
+var keyErrors = ['-', '+', 'e', 'E', '.'];
```
Comment (alexmfritz) [inline src/main.js:24]: Dem pesky keys!

File: src/main.js [hunk 3]
```diff
@@ -14,16 +14,29 @@ var timerView = document.querySelector('.timer-presenter');
 var userActivityInputView = document.querySelector('.user-activity-input');
 //inputs
 var userAccomplishInput = document.querySelector('#user-accomplish');
-var userMinutesInput = document.querySelector('#minutes');
-var userSecondsInput = document.querySelector('#seconds');
+var userMinutesInput = document.querySelector('#user-minutes');
+var userSecondsInput = document.querySelector('#user-seconds');
```
Comment (devonwigle) [inline src/main.js:18]: We should check the rest of our variables as well, since these were renamed.

File: src/main.js [hunk 4]
```diff
@@ -14,16 +14,29 @@ var timerView = document.querySelector('.timer-presenter');
...
 //errors
 var selectionError = document.querySelector('.selection');
 var accomplishError = document.querySelector('.accomplish');
 var minutesError = document.querySelector('.minutes');
 var secondsError = document.querySelector('.seconds');
+var keyErrors = ['-', '+', 'e', 'E', '.'];
 
 //event listeners
 
+userMinutesInput.addEventListener('keydown', function(event) {
```
Comment (alexmfritz) [inline src/main.js:28]: "keydown" === bleh

**Review by devonwigle — APPROVED** [review]

File: src/main.js [hunk 5]
```diff
@@ -75,6 +83,16 @@ function highlight(element, element2, rule, icon) {
   }
 };
 
+function unhighlight(element, element2, rule, icon) {
```
Comment (alexmfritz) [inline src/main.js:86]: I wonder if we can refactor this function to not include a conditional?

**Pull request merged**